skrf/network.py

```diff
@@ -2963,6 +3095,45 @@ def connect(ntwkA, k, ntwkB, l, num=1):
         ntwkC.renumber(from_ports=from_ports,
                        to_ports=to_ports)
 
+    # if ntwkA and ntwkB are both 2port, and either one has noise, calculate ntwkC's noise
+    if num == 1 and ntwkA.nports == 2 and ntwkB.nports == 2 and (ntwkA.noisy or ntwkB.noisy):
```

Comment:
when you add a new property to the class and want to test for it,  you should except an AttributeError. otherwise any  old pickled networks will fail if they are loaded with the new code. 
i dont know how many people use pickling to store their data.

Reply:
Fixed I think!